src/gpuinfo.jl

```diff
@@ -201,16 +202,16 @@ function gpuinfo_p2p_access()
             end
         end
 
-        printstyled("P2P Access Supported:\n"; bold=true)
-        display(mat_p2p_access_supported)
-        println()
+        printstyled(io,"P2P Access Supported:\n"; bold=true)
+        show(io, "text/plain",mat_p2p_access_supported)
+        println(io)
         if mat_p2p_access_supported != mat_p2p_can_access
-            printstyled("P2P Can Access:\n"; bold=true)
-            display(mat_p2p_can_access)
-            println()
+            printstyled(io,"P2P Can Access:\n"; bold=true)
+            show(io, "text/plain",mat_p2p_can_access)
+            println(io)
         end
-        printstyled("P2P Atomic Supported:\n"; bold=true)
-        display(mat_p2p_atomic_supported)
+        printstyled(io,"P2P Atomic Supported:\n"; bold=true)
+        show(io, "text/plain",mat_p2p_atomic_supported)
```

Comment:
I would add one more `println(io)` after each of the matrix printing. I believe that `display` implicitly did that. Currently it looks a it cramped.